app/borrow/components/__tests__/terms-row.test.tsx

```
@@ -0,0 +1,119 @@
+    assetIsIntentsAsset: false,
+    depositAmount: null,
+    hasEnoughMtBalance: false,
+  }),
+}));
+
+vi.mock("@/config/universal-account", () => ({
+  isUniversalAuthMethod: () => false,
+}));
+
+vi.mock("@/app/sign-in-button", () => ({
+  SignInButton: () => <button>Sign In</button>,
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)